src/lib/Microsoft.Health.Events/EventConsumers/Service/EventBatchingService.cs

```diff
@@ -80,6 +81,11 @@ public async Task ConsumeEvent(IEventMessage eventArg)
                     var windowThresholdTime = GetPartition(partitionId).GetPartitionWindow();
                     await ThresholdWaitReached(partitionId, windowThresholdTime);
                 }
+                else
+                {
+                    // If we received the timer event and there are no enqueued events in the partition, then simply update the data freshness.
+                    LogDataFreshness(partitionId, triggerReason: nameof(ThresholdWaitReached));
```

Comment:
This is basically so that on a glance, when the argument provided is not obvious as to what param it is being passed in for, it is easier to interpret by using the field name too. Eg: its is clear from the naming that 'partitionId' arg is referring to 'partitionId', but if I passed in a 1,2,etc (or similarly for bool params, if I just passed true/false). it wouldn't be clear what this integer is indicative of unless I go the function definition and look it up. So just to make interpretation easier, its a coding guideline that is suggested.